test/unittests/statetest_loader_tx_test.cpp

```diff
@@ -16,6 +16,7 @@ TEST(statetest_loader, tx_create_legacy)
     constexpr std::string_view input = R"({
         "input": "b0b1",
         "gas": "0x9091",
+        "chainId": "0x1",
```

Comment:
Value 1 is default is some places so this is not a good test.

Reply:
default value for `chain_id` is 0. I can change to something different then 0 or 1, but in this case we use only from_json which reads it as it's in json and doesn't change default value.